lib/matplotlib/tests/test_colorbar.py

```diff
@@ -327,6 +327,35 @@ def test_colorbar_minorticks_on_off():
     np.testing.assert_equal(cbar.ax.yaxis.get_minorticklocs(), [])
 
 
+def test_cbar_minorticks_for_rc_xyminortickvisible():
+    """
+    issue gh-16468.
+
+    Making sure that minor ticks on the colorbar are turned on
+    (internally) using the cbar.minorticks_on() method when
+    rcParams['xtick.minor.visible'] = True (for horizontal cbar)
+    rcParams['ytick.minor.visible'] = True (for vertical cbar).
+    Using cbar.minorticks_on() ensures that the minor ticks
+    don't overflow into the extend regions of the colorbar.
+    """
+
+    plt.rcParams['ytick.minor.visible'] = True
+    plt.rcParams['xtick.minor.visible'] = True
+    np.random.seed(seed=12345)
+    x = np.random.randn(20, 20)
+    vmin, vmax = -2.3, 3.3
+    fig, ax = plt.subplots()
+    im = ax.pcolormesh(x, vmin=vmin, vmax=vmax)
```

Comment:
You don't need to put in random data (see also https://github.com/matplotlib/matplotlib/pull/16438).  This would actually be shorter and easier to read with e.g. `im = ax.pcolormesh([[1, 2]], vmin=0.5, vmax=2.5)` (or whatever suitable values).

Reply:
Thanks. Pushed a change.